nomad/utils/text.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+import re
+import unicodedata
+
+
+_punct_re = re.compile(r'[\t !"#$%&\'()*\-/<=>?@\[\\\]^_`{|},.]+')
+
+
+def slugify(text):
+    """ Returns a ASCII-only slug version of the considered text. """
+    result = []
+    for word in _punct_re.split(text.lower()):
+        word = unicodedata.normalize('NFKD', word).encode('ascii', 'ignore').decode('ascii')
```

Comment:
I'm uneasy about this `ignore` flag. Since the project name's only purpose is to help determine a container name, what about enforcing (and documenting) a 7bit ASCII range? I don't see any reason to go beyond that range and I feel like this may cause some unforeseen problems.